ut-robomaster/src/subsystems/agitator/agitator_subsystem.cpp

```diff
@@ -70,4 +74,14 @@ float AgitatorSubsystem::getShapedVelocity(float time, float a, float phi, float
 void AgitatorSubsystem::setBallsPerSecond(float bps) { ballsPerSecond = bps; }
 float AgitatorSubsystem::getPosition() { return agitator.measurePosition(); }
 float AgitatorSubsystem::getVelocity() { return agitator.measureVelocity(); }
+
+bool AgitatorSubsystem::hardwareOk()
+{
+#ifdef TARGET_HERO
+    return flywheel->hardwareOk() && agitator.isOnline() && feeder.isOnline();
+#else
+    return flywheel->hardwareOk() && agitator.isOnline();
```

Comment:
Lets keep the Agitator hardware check to Agitator hardware only and check the flywheels separately with the `flywheel->isActive()` on line 57